docs/Available-Symbols-Generators.md

```diff
@@ -182,42 +183,17 @@ Uses constant value.
 [`Sample`](https://github.com/dotnet/dotnet-template-samples/tree/master/13-constant-value)
 
 
-## Evaluate
-
-Defines expression that will be evaluated during the processing of the template based on parameters and other symbols.
-The `computed` type symbols can be use for same purpose.
-
-#### Parameters
-
-| Name     |Data Type| Description   |
-|----------|------|---------------|
-|value|string|expression to be evaluated| 
-
-### Samples
-
-In this sample `IndividualAuth` is `true` if the value of `auth`, another symbol defined in the template, is `IndividualB2C`
-```
-    "IndividualAuth": {
-      "type": "generated",
-	  "generator": "evaluate",
-	  "parameters": {
-		  "action": "(auth == \"IndividualB2C\")"
-		},
-    },
-```
-### Related
-[`Implementation class`](https://github.com/dotnet/templating/blob/main/src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Macros/EvaluateMacro.cs)
-
-
 ## Port
 Gets an available port number on the machine.   
 During evaluation looks for a valid free port number trying to create a socket, and in case of problems, returns the value defined in the `fallback` parameter.
 
 #### Parameters
 
-| Name     |Data Type| Description   |
-|----------|------|---------------|
-|fallback|string|fallback value| 
+|Name|Data Type|Description|Mandatory|
+|----------|------|---------------|---|
+|`high`|`integer`|defined the high bound of range to select port from|no, default: `1024`|
+|`low`|`integer`|defined the low bound of range to select port from|no, default: `65535`|
```

Comment:
high and low defaults switched here

Reply:
IIRC the bounds are forces by the code (e.g. if low is specified to be below 1024, it is silently adjusted to 1024) - if it's so, it's worth documenting. e.g. "The low bound of range for port selection. Will be adjusted to range <1024; 65535> if specified outside of this range."

Reply:
also added reserved ports